Dockerfile

```diff
@@ -95,7 +95,9 @@ ENV S6_BEHAVIOUR_IF_STAGE2_FAILS="2" \
   LIBRENMS_DOCKER="1" \
   TZ="UTC" \
   PUID="1000" \
-  PGID="1000"
+  PGID="1000" \
+  MIBDIRS="/opt/librenms/mibs:/opt/librenms/mibs/cisco" \
```

Comment:
I think `MIBDIRS` should be outsourced from the Dockerfile and placed in your `08-svc-snmptrapd.sh`. We should also allow users to provide their own MIBs. `/data/mibs` could be a good place for this.

Reply:
Also maybe allow additional dirs with an env var like `ADDITIONAL_MIBS` and append them to `MIBDIRS`.